utils/txHandler.ts

```diff
@@ -155,6 +155,26 @@ export const getEventResultFromMangataTx = function (
   return createEventResultfromExtrinsic(extrinsicResult as MangataGenericEvent);
 };
 
+export async function getEventErrorfromSudo(sudoEvent: MangataGenericEvent[]) {
+  const api = getApi();
+
+  const userBootstrapErr = hexToU8a(
+    //@ts-ignore
+    sudoEvent[0].event.data[0].asErr.value.error.toString()
+  );
+
+  const userBootstrapIndex =
+    //@ts-ignore
+    sudoEvent[0].event.data[0].asErr.value.index.toString();
```

Comment:
looks like you are only expecting the error on the first element `sudoEvent[0]` , could you perhaps iterate over the whole list and return the errors?

Reply:
We have only one raw in sudoEvent, that's why I use only `sudoEvent[0]`